textproto/mod.ts

```diff
@@ -119,7 +119,11 @@ export class TextProtoReader {
       }
       let value = str(kv.subarray(i));
 
-      m.append(key, value);
+      // In case of invalid header we swallow the error
+      // example: "Audio Mode" => invalid due to space in the key
+      try {
+        m.append(key, value);
+      } catch {}
```

Comment:
Are there other exceptions that could happen?

Reply:
You mean during the parsing process? No. Here it's just to swallow invalid headers, invalid values will be catch during previous steps.